ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -683,16 +683,20 @@ func countAPFDevices() int {
 
 func postInitAppScript() string {
 
-	postInitAppScriptStr := `PORT0_SETUP_SCRIPT=/work/scripts/port0-setup.sh
-PORT0_SETUP_LOG=/work/port0-setup.log
+	postInitAppScriptStr := `PORT_SETUP_SCRIPT=/work/scripts/port-setup.sh
+PORT_SETUP_LOG=/work/port-setup.log
 
-pkill -9 $(basename ${PORT0_SETUP_SCRIPT})
-/bin/rm -f ${PORT0_SETUP_SCRIPT} ${PORT0_SETUP_LOG}
+pkill -9 $(basename ${PORT_SETUP_SCRIPT})
+/bin/rm -f ${PORT_SETUP_SCRIPT} ${PORT_SETUP_LOG}
 
-cat<<PORT0_CONFIG_EOF > ${PORT0_SETUP_SCRIPT}
+#To ensure files get removed.
+sync
+
+cat<<PORT_CONFIG_EOF > ${PORT_SETUP_SCRIPT}
 #!/bin/bash
-IDPF_VPORT_NAME="enp0s1f0d4"
-ACC_VPORT_ID=0x4
+set -x
+IDPF_VPORT_NAME="enp0s1f0d5"
```

Comment:
Nit: Do you think it is possible to use const variables for this and the next line to fill this up using `%s` ?
The variable `PHY_PORT1_INTF_INDEX` is already present at the top of this doc. We can also add another const variable for the Post representor too

Reply:
Ok, I started to do this using fmt.Sprintf, but this turned out to be more work than expected, due to internal %x shell format specifier. Can be done as part of a separate PR(later).